<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtVTKViewerWidget</class>
 <widget class="QWidget" name="QtVTKViewerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QToolBox" name="toolBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>2</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>155</width>
        <height>505</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Image Widget</string>
      </attribute>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>201</width>
         <height>246</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QCheckBox" name="checkBoxShowPlanes">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Show Planes</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="tristate">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0" colspan="2">
           <widget class="QCheckBox" name="checkBoxShowImageActor">
            <property name="text">
             <string>Show Image Actor</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>18</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="1">
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="labelx">
              <property name="text">
               <string>X  slice =</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="2">
             <widget class="QLineEdit" name="lineEditX"/>
            </item>
            <item row="1" column="0" colspan="3">
             <widget class="QSlider" name="horizontalSliderX">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Y slice =</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="2" column="2">
             <widget class="QLineEdit" name="lineEditY"/>
            </item>
            <item row="3" column="0" colspan="3">
             <widget class="QSlider" name="horizontalSliderY">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Z  slice =</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="4" column="2">
             <widget class="QLineEdit" name="lineEditZ"/>
            </item>
            <item row="5" column="0" colspan="3">
             <widget class="QSlider" name="horizontalSliderZ">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>155</width>
        <height>505</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Volume Rendering</string>
      </attribute>
      <widget class="QtVolumeRendererPanel" name="m_VolumeRender" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>181</width>
         <height>221</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>10</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>155</width>
        <height>505</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Surface Rendering</string>
      </attribute>
      <widget class="QtSurfaceRenderer" name="m_SurfaceRender" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>221</width>
         <height>381</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QtVTKViewer" name="_qtvtkviewer" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>8</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <zorder>toolBox</zorder>
     <zorder>toolBox</zorder>
     <zorder>toolBox</zorder>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtVTKViewer</class>
   <extends>QWidget</extends>
   <header>qtvtkviewer.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QtVolumeRendererPanel</class>
   <extends>QWidget</extends>
   <header>qtvolumerendererpanel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QtSurfaceRenderer</class>
   <extends>QWidget</extends>
   <header location="global">qtsurfacerenderer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>checkBoxShowImageActor</sender>
   <signal>clicked(bool)</signal>
   <receiver>horizontalSliderX</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>81</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>81</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxShowImageActor</sender>
   <signal>clicked(bool)</signal>
   <receiver>horizontalSliderY</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>128</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>124</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxShowImageActor</sender>
   <signal>clicked(bool)</signal>
   <receiver>horizontalSliderZ</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>87</x>
     <y>297</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
